libs/community/langchain_community/output_parsers/openai_functions.py

```diff
@@ -0,0 +1,174 @@
+import copy
```

Comment:
we want to keep this in LangChain

openai is becoming a standard-ish interface that more than one provider is implementing, so we will likely keep utils for openai in langchain

Reply:
It is a stab in the whole idea of the `community` package. Meta, Alibaba, Baidu, Google, Antrophic would not be happy by this, no? LangChain has 5M downloads/month, it is a lot of leverage that should not be easily passed to one partner company.
The second reason is a more important one. There will be another better model from a different company in several months. "Standard" should be used for something more stable.

Reply:
their interface is adopted by many other model providers - there are many "open ai drop ins", the meta/anthropic/etc interface is not

in the future (langchain 0.9.0 or whatnot) we can update what interfaces are the standard, but for now, openai function calling is a standard

Reply:
Yeah, OpenAI functions are almost a standard. Not sure about OpenAI tools.